config/detekt-baseline.xml

```diff
@@ -313,6 +313,14 @@
     <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onLongPress(hitResult: HitResult)</ID>
     <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaAdded(media: Media)</ID>
     <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaRemoved(media: Media)</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaActivated(mediaSessionController: MediaSession.Controller)</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaDeactivated()</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaMetadataChanged( mediaSessionController: MediaSession.Controller, metadata: MediaSession.Metadata )</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaFeatureChanged( mediaSessionController: MediaSession.Controller, features: MediaSession.Feature )</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaPlaybackStateChanged( mediaSessionController: MediaSession.Controller, playbackState: MediaSession.PlaybackState )</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaPositionStateChanged( mediaSessionController: MediaSession.Controller, positionState: MediaSession.PositionState )</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaMuteChanged( mediaSessionController: MediaSession.Controller, muted: Boolean )</ID>
+    <ID>UndocumentedPublicFunction:EngineSession.kt$EngineSession.Observer$fun onMediaFullscreenChanged( mediaSessionController: MediaSession.Controller, fullscreen: Boolean, elementMetadata: MediaSession.ElementMetadata? )</ID>
```

Comment:
Could we just document them instead of adding more to the baseline file? That file was only needed to get us started. :)

Reply:
Sure, I"ll document them :)